R/import_methods.R

```diff
@@ -124,44 +124,82 @@ import_delim <- function(file, which = 1, sep = "auto", header = "auto", strings
 }
 
 #' @export
-.import.rio_r <- function(file, which = 1, ...) {
+.import.rio_r <- function(file, which = 1, trust = TRUE, ...) {
+    lifecycle::deprecate_warn(
+      when = "2.0.0",
+      what = "import(trust)",
```

Comment:
By setting up it this way, the `deprecate_warn()` will still emit the warning when trust is `FALSE`.

Another way to handle this is probably like this:

```r
.import.rio_r <- function(file, which = 1, trust, ...) {
    if (missing(trust)) {
        lifecycle::deprecate_warn(
                       when = "2.0.0",
                       what = "import(trust)",
                       details = "Missing `trust` will be set to FALSE by default for rdata, RDS, and dump.")
        trust <- TRUE
    }
}
```

Can add this to the below mentioned helper function, so that we don't need to update several methods when we are dealing with v2.0.0.